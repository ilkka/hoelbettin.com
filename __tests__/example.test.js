import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

import Lander from '../pages/lander.js';

describe('With Enzyme', () => {
  it.skip('Lander has hero image', () => {
    // TODO: does not work yet lol don't know why
    const lander = shallow(<Lander />);

    expect(lander.find('img').to.have.length(1));
  });
});

describe('With Snapshot Testing', () => {
  it('Lander has hero image', () => {
    const component = renderer.create(<Lander />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

