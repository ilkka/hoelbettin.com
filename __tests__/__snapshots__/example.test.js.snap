// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Lander has hero image 1`] = `
<div>
  <div>
    <div
      className="hero-image"
    >
      <img
        alt="HÃ¶elbettin OG"
        className="lander__HeroImage-y7a1d1-0 jHenZq"
        src="/static/logo.jpg"
      />
    </div>
  </div>
</div>
`;
